tests/config/config.travis-ci.php

```diff
@@ -2,7 +2,7 @@
 
 $LRSs = [
     [
-        'endpoint' => 'https://cloud.scorm.com/tc/0CKX3A0SF2/sandbox/',
+        'endpoint' => 'https://cloud.scorm.com:443/tc/0CKX3A0SF2/sandbox/',
```

Comment:
Yep. The purpose is to get [this line](https://github.com/RusticiSoftware/TinCanPHP/blob/3aa60575bdca35de7667ab5dd987a7211a5e8337/src/RemoteLRS.php#L1134) to execute.